en/docs/develop/scriptLibrary-rest-api.md

```diff
@@ -0,0 +1,37 @@
+---
+template: templates/swagger.html
+---
+??? Note "Note: Click to view"
+    Do the following to try out the REST APIs with your local instance of WSO2 Identity Server.
```

Comment:
Follow the steps given below to try out the REST APIs with your local instance of WSO2 Identity Server.